src/main/java/uk/gov/hmcts/reform/professionalapi/domain/OrganisationStatus.java

```diff
@@ -1,8 +1,18 @@
 package uk.gov.hmcts.reform.professionalapi.domain;
 
 public enum OrganisationStatus {
-    PENDING,
-    ACTIVE,
-    BLOCKED,
-    DELETED
+    PENDING("pending"),
```

Comment:
I don't agree with this change. There is no need to store values  in the enum when the name is going to perfectly match the name of the enum. You should be ignoring the case anyhow in the controller and always performing a .Upper() on both string before comparing them.

Reply:
Actions:
1. Please remove the status member
2. Update to code to use .name() instead of .getStatus
3. Never hard-code any strings such as "PENDING" or "pending" unless unit testing the enum class itself of course.

Reply:
Removed the status member and used .name() method instead of .getStatus().